packages/commitlint-config/package.json

```diff
@@ -18,6 +18,6 @@
     "@commitlint/config-conventional": "^12.0.1"
   },
   "peerDependencies": {
-    "@commitlint/cli": "^8.3.5"
+    "@commitlint/cli": "^8.3.5 || ^12.0.0"
```

Comment:
If we're saying we support v8 & 12, shouldn't we do a range so we include everything between >=8 && <13?

Reply:
Updated